docs/hxapi/carhire/availability/index.html

```diff
@@ -61,15 +61,15 @@ <h2 id="request">Request</h2>
 </code></pre></div></div>
 
 <p>Example request:</p>
-<ul class="uk-tab" data-uk-switcher="{connect:'#d0c793d2-e209-4e26-8320-bf886369439d'}">
+<ul class="uk-tab" data-uk-switcher="{connect:'#06de44e9-75cd-413f-b4ec-73cb9826f15e'}">
```

Comment:
what is this uuid?

Reply:
all of these are automatically generated by the build process when you commit